routemaster/client.rb

```diff
@@ -59,6 +60,20 @@ def subscribe(options = {})
       end
     end
 
+    def monitor_topics
+      response = _get('/topics') do |r|
+        r.headers['Content-Type'] = 'application/json'
+      end
+
+      unless response.success?
+        raise 'failed to connect to /topics'
+      end
```

Comment:
If I do this, I need to update the rest of the file IMHO so it keeps consistent. What do you think ?